packages/devextreme/testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/toolbarAiDialogIntegration.tests.js

```diff
@@ -0,0 +1,107 @@
+import $ from 'jquery';
+
+import 'ui/html_editor';
+
+import { openAiDialog } from '../../../helpers/aiToolbarMenu.js';
+import { selectInsertionMode, setResultText, getResultText } from '../../../helpers/aiDialog.js';
+
+const setupHtmlEditorWithAi = (onValueChanged, config) => {
```

Comment:
Why is `onValueChanged` so special to be passed as a separate first parameter?
It's confusing to read it in a place of using